extension/docs/Step1-getPaymentMethods.md

```diff
@@ -0,0 +1,109 @@
+## Step 1: Get available payment methods
+When your shopper is ready to pay, get a list of the available payment methods based on their country, device, and the payment amount.
+
+From the merchant server, [Create/Update CTP payment](https://docs.commercetools.com/http-api-projects-payments#create-a-payment) with `getPaymentMethodsRequest` custom field.  
+
+### Request
+
+Set `getPaymentMethodsRequest` custom field for a shopper in the Germany, for a payment of `10 EUR`: 
+
+``` json
+{
+  "countryCode": "DE",
+  "shopperLocale": "de-DE",
+  "amount": {
+    "currency": "EUR",
+    "value": 1000
+  }
+}
+```
+
+CTP payment representation:
+
+``` json
+{
+  "amountPlanned": {
+    "currencyCode": "EUR",
+    "centAmount": 1000
+  },
+  "paymentMethodInfo": {
+    "paymentInterface": "ctp-adyen-integration"
+  },
+  "custom": {
+    "type": {
+      "typeId": "type",
+      "key": "ctp-adyen-integration-web-components-payment-type"
+    },
+    "fields": {
+      "getPaymentMethodsRequest": "{\"countryCode\":\"DE\",\"shopperLocale\":\"de-DE\",\"amount\":{\"currency\":\"EUR\",\"value\":1000}}"
+    }
+  }
+}
+```
+
+### Response
+
+Pass the `getPaymentMethodsResponse` to your front end. You might use this in the next step to show which payment methods are available for the shopper.
+
+The response includes the list of available paymentMethods:
+
+``` json
+{
+ "paymentMethods":[
+  {
+    "details":[...],
+    "name":"Credit Card",
+    "type":"scheme"
+    ...
+  },
+  {
+    "details":[...],
+    "name":"SEPA Direct Debit",
+    "type":"sepadirectdebit"
+  },
+  ...
+  ]
+}
+```
+
+CTP payment representation:
```

Comment:
I think we can remove unnecessary data from the representation which is not focus of this step. For example: amountPlanned or interfaceInteractions.